#! /bin/sh /usr/share/dpatch/dpatch-run
## 02.htlatex_documentation.dpatch by Piotr Roszatycki <dexter@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Minor doc fixes.

@DPATCH@
diff -urNad lyx-1.3.6/lib/doc/de_Extended.lyx /tmp/dpep.ksPjwu/lyx-1.3.6/lib/doc/de_Extended.lyx
--- lyx-1.3.6/lib/doc/de_Extended.lyx	2004-05-11 01:22:24.000000000 +1000
+++ /tmp/dpep.ksPjwu/lyx-1.3.6/lib/doc/de_Extended.lyx	2005-09-13 16:04:32.869410560 +1000
@@ -13975,7 +13975,7 @@
 
 LyX-Dokumente können nach HTML konvertiert werden, indem sie zunächst nach
  LaTeX, dann nach HTML konvertiert werden.
- LyX kennt zur Zeit drei LaTeX 
+ LyX kennt zur Zeit vier LaTeX 
 \begin_inset Formula $\rightarrow$
 \end_inset 
 
@@ -13988,10 +13988,14 @@
 latex2\SpecialChar \-
 html
 \family default 
- und 
+, 
 \family typewriter 
 hevea
 \family default 
+ und 
+\family typewriter 
+htlatex
+\family default 
 .
  
 \family typewriter 

